modules/installation-uninstall-clouds.adoc

```diff
@@ -156,9 +159,11 @@ If your cluster uses Technology Preview features that are enabled by the `TechPr
 --
 endif::ibm-cloud,ibm-power-vs[]
 
-. Optional: Delete the `<installation_directory>` directory and the
-{product-title} installation program.
+ifdef::osp[]
+. Use the `coreos-installer` tool to add the `coreos.inst.wipe=yes` flag to the Preboot Execution Environment (PXE) boot configuration. This operation wipes the disk on your system so that if you create a new cluster, you have a clean installation environment. For more detailed instructions, see link:https://access.redhat.com/solutions/7128657[How to wipe OpenStack disks in {product-title} 4 reinstallation] (Knowledgebase article).
```

Comment:
Disk wiping isn’t always necessary. It depends on the storage setup.

What do you think about using phrases like "Consider using..." to make it clear that this is optional guidance?

Reply:
Thanks, Itay. I marked this step as optional.